File: be_laravel/app/Models/FreeAvatar.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class FreeAvatar extends Model
{
    use HasFactory;

    public $timestamps = true;
    protected $table = 'free_avatars';
    protected $primaryKey = 'id_freeavatar';

    protected $fillable = [
        'id_freeavatar',
        'photo_freeavatar',
    ];

    public static function getAllData()
    {
        return self::all();
    }

    public static function getFreeAvatarById($id)
    {
        return self::find($id);
    }
}

File: be_golang/models/free_avatar.go
package models

import "time"

type FreeAvatar struct {
	IdFreeavatar    int       `json:"id_freeavatar" gorm:"primaryKey:autoIncrement"`
	PhotoFreeavatar string    `json:"photo_freeavatar" gorm:"type:varchar(255)"`
	CreatedAt       time.Time `json:"created_at"`
	UpdatedAt       time.Time `json:"updated_at"`
}
